src/render.js

```diff
@@ -115,7 +182,15 @@ const render = async function(jsPackages, props, globals, requestStats) {
 
     try {
         // If Apollo is required, get it configured on the context.
-        configureApolloNetwork(context.window);
+        const apolloNetwork: ?ApolloNetworkConfiguration = (context.window
+            .ApolloNetwork: any);
+        const apolloGlobals = configureApolloNetwork(apolloNetwork);
+
+        /**
```

Comment:
Is this new? I can't find where this moved from.

Reply:
It was inside the configure method before. Nothing new